packages/Badge/src/Badge.spec.tsx

```diff
@@ -9,13 +9,22 @@ describe('packages/Badge', () => {
   const className = 'test-pill-class';
   const child = 'Bubble Pill';
 
+  function isElement(node: Node): node is Element {
+    return node.nodeType === Node.ELEMENT_NODE;
+  }
+
   const { container } = render(
     <Badge className={className} onClick={onClick}>
       {child}
     </Badge>,
   );
 
   const badge = container.firstChild;
+
+  if (badge == null || !isElement(badge)) {
```

Comment:
Can this just be a `test()`? Or is this needed for TS?

Reply:
Needed for TS type narrowing (discussed in Slack)